examples/webgpu_sandbox.html

```diff
@@ -1,36 +1,33 @@
 <html lang="en">
 	<head>
-		<title>three.js - WebGPU - Sandbox</title>
+		<title>three.js webgpu - sandbox</title>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
 		<link type="text/css" rel="stylesheet" href="main.css">
 	</head>
 	<body>
 
 		<div id="info">
-			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> WebGPU - Sandbox
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> webgpu - sandbox
 		</div>
 
 		<script type="importmap">
 			{
 				"imports": {
-					"three": "../build/three.module.js",
-					"three/addons/": "./jsm/",
-					"three/nodes": "./jsm/nodes/Nodes.js"
+					"three": "../build/three.webgpu.js",
+					"three/debug": "../src/Three.WebGPU.js",
```

Comment:
`debug` does not appear to be needed. Should we have an example in which it is?

Reply:
I purposely put it in `sandbox` because I thought it could convey the idea of ​​`debug` since I can't make simple comments in the JSON, maybe there is a more suitable way to do this...

Reply:
Your choice... I'd vote to merge this PR ASAP if @Mugen87 approves.

Reply:
Let me think about the naming for all this for a couple of days 🙏

Reply:
I replaced `three/nodes` to `three/webgpu`. I imagined that developers could use three alias in vite like:

```js
// vite.config.js

import { defineConfig } from 'vite';

export default defineConfig({
  resolve: {
    alias: {
      'three': 'three/webgpu'
    }
  }
});
```